roles/docker/tasks/set_facts_dns.yml

```diff
@@ -40,13 +40,13 @@
 
 - name: add system nameservers to docker options
   set_fact:
-    docker_dns_servers: "{{ docker_dns_servers + [item] }}"
-  with_items: "{{ system_nameservers.stdout_lines|default([]) }}"
+    docker_dns_servers: "{{ docker_dns_servers | system_nameservers.stdout_lines | unique }}"
```

Comment:
Now we have unique in name servers as well...that will mess up some systems as it may change order.